src/components/Color.tsx

```diff
@@ -0,0 +1,84 @@
+import React, {FC, ReactNode} from 'react';
+import PropTypes from 'prop-types';
+import arrify from 'arrify';
+import chalk, {Chalk} from 'chalk';
+import {Except} from 'type-fest';
+
+const methods = [
+	'hex',
+	'hsl',
+	'hsv',
+	'hwb',
+	'rgb',
+	'keyword',
+	'bgHex',
+	'bgHsl',
+	'bgHsv',
+	'bgHwb',
+	'bgRgb',
+	'bgKeyword',
+	'ansi',
+	'ansi256',
+	'bgAnsi',
+	'bgAnsi256'
+];
+
+/**
+ * The `<Color>` compoment is a simple wrapper around the `chalk` API. It supports all of the `chalk`'s methods as `props`.
+ */
+export const Color: FC<ColorProps & {children: ReactNode}> = ({
```

Comment:
I think `React.FC` type already includes a type for `children` prop, you don't need it to include it here. It can just be `FC<ColorProps>`.

Reply:
TypeScript fails to compile with errors because over defaultValue and propTypes. I can remove propTypes and defaultValue and children or keep the children. I would prefer to remove propTypes.

<img width="675" alt="image" src="https://user-images.githubusercontent.com/74687/77829007-8fcb8680-70f5-11ea-9af6-c6fde73b24c3.png">

Reply:
What's the error that is being reported here? It's not visible in the screenshots.

Reply:
@vadimdemedes if I removed children props, I get a typescript error on propTypes - you can see it at the bottom of the first screenshot. 

Since, FC type has children as optional by default, I think the best would be to add children to ColorProps type because children are not optional.

Reply:
I meant the error message from TypeScript, it's not in the screenshot.

If it works with `children` in `ColorProps` while still keeping `propTypes`, if I understood correctly, that'd be great.